tests/events_test.py

```diff
@@ -15,52 +15,57 @@ def test_create_event_should_return_event(self):
         events = Events()
         event = events.create_event("foobar", None, ReachabilityEvent)
 
-        assert event.id > 0
+        assert isinstance(event, ReachabilityEvent)
 
-    def test_event_registry_should_contain_one_event_when_first_event_is_created(self):
+    def test_event_registry_should_not_contain_newly_created_event(self):
```

Comment:
This by itself doesn't make it clear for me why it should not be in the event registry, maybe something like this is clearer:

```suggestion
    def test_event_registry_should_not_contain_uncommited_event(self):
```

Reply:
Much better - fixed now. Thanks!